lib/tests/streamlit/legacy_data_frame_test.py

```diff
@@ -272,415 +262,3 @@ def test_marshall_any_array(self):
 
         with pytest.raises(NotImplementedError, match="^Dtype <U6 not understood.$"):
             data_frame._marshall_any_array(str_data, str_proto)
-
-    def test_add_rows(self):
-        """Test streamlit.data_frame.add_rows."""
```

Comment:
All these removed tests are for functionality that no longer exists (the actual meat of the `add_rows` implementation).